### PR TITLE
feat: BOS for Ethereum developers doc

### PR DESCRIPTION
**Pull request by zavodil**

### CODE REVIEW

**Comment by bucanero** [comment]
@zavodil , since this is a new article, I suggest to edit `website/sidebar.json` too and add the article there, so it shows up in the left navigation

**Comment by bucanero** [comment]
> @bucanero will do. Do you already have a category like "Building on BOS"? If no, what is the best place to put it?

yes, we have a section for BOS here: https://docs.near.org/develop/integrate/welcome

I think the best option would be to replace the following "Basic ether.js" article:
https://docs.near.org/bos/tutorial/ethers-js

since that old article feels like a place holder, your new article would be much better for devs to learn how to use BOS+Etherjs

File: website/sidebars.json [hunk 1]
```diff
@@ -618,7 +618,7 @@
         "bos/tutorial/using-iframes",
         "bos/tutorial/bos-gateway",
         "bos/tutorial/push-notifications",
-        "bos/tutorial/ethers-js",
+        "bos/tutorial/bos-ethersjs",
```
Comment (bucanero) [inline website/sidebars.json:621]:
```suggestion
        "bos/tutorial/ethers-js",
```

File: docs/bos/tutorial/bos-ethersjs.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
+---
+id: bos-ethersjs
```
Comment (bucanero) [inline docs/bos/tutorial/bos-ethersjs.md:2]:
```suggestion
id: ethers-js
```

**Comment by bucanero** [comment]
@zavodil , could you remove the old file `hello-ethersJS.md`? thank you! 🙏

**Comment by zavodil** [comment]
@bucanero done

**Pull request merged**